nginx-controller/nginx/configurator.go

```diff
@@ -380,6 +389,30 @@ func (cnf *Configurator) createConfig(ingEx *IngressEx) Config {
 		ingCfg.Ports = ports
 	}
 
+	ingCfg.Enabled = true
+	if listenAddress, exists := ingEx.Ingress.Annotations["nginx.org/listen-address"]; exists {
+		ingCfg.Enabled = false
+
+		addrs, err := net.InterfaceAddrs()
+		if err != nil {
+			glog.Errorf("unable to list interface addresses: %s", err)
+		} else {
+			listenIP := net.ParseIP(listenAddress)
+			if err != nil {
+				glog.Errorf("unable to parse listen address: %s", err)
+			} else {
+				for _, addr := range addrs {
+					hostIP, _, _ := net.ParseCIDR(addr.String())
```

Comment:
why do you ignore `error` of ParseCIDR?

Reply:
Because this address is coming from `net.InterfaceAddrs()` function, which will always be correct address.